src/theory/datatypes/theory_datatypes.cpp

```diff
@@ -308,15 +307,6 @@ void TheoryDatatypes::postCheck(Effort level)
                   }
                 }
               }
-              //if we want to force an assignment of constructors to all ground eqc
-              //d_dtfCounter++;
-              if (!needSplit && options().datatypes.dtForceAssignment
```

Comment:
Are you planning to run benchmarks?

Reply:
This does not change the behavior.  The removed option was highly experimental.